concordium-consensus/src/Concordium/Scheduler.hs

```diff
@@ -2750,12 +2750,42 @@ handleChainUpdate (WithMetadata{wmdData = ui@UpdateInstruction{..}, ..}, mVerRes
                                 ValidatorScoreParametersUpdatePayload u -> case sIsSupported SPTValidatorScoreParameters scpv of
                                     STrue -> checkSigAndEnqueue $ UVValidatorScoreParameters u
                                     SFalse -> return $ TxInvalid NotSupportedAtCurrentProtocolVersion
+                                CreatePLTUpdatePayload payload -> case sIsSupported SPTProtocolLevelTokensParameters scpv of
+                                    SFalse -> return $ TxInvalid NotSupportedAtCurrentProtocolVersion
+                                    STrue ->
+                                        if updateEffectiveTime uiHeader /= 0
+                                            then return $ TxInvalid InvalidUpdateTime
+                                            else do
+                                                -- TODO Check signature when relevant update keys are introduced.
```

Comment:
Add a link to the ticket in the TODO (i.e. https://linear.app/concordium/issue/NOD-701/add-authorization-for-createplt-chain-updates). This helps us be sure it won't slip through the cracks (because there's a ticket), helps document the TODO and can help find the relevant code when implementing the ticket.